doc/releases/changelog-dev.md

```diff
@@ -206,6 +206,10 @@
   array([1.-0.j, 0.-1.j])
   ```
 
+* The `ctrl` transform and `ControlledOperation` have been moved to the new `qml.ops.op_math`
+  submodule.  The developer-facing `ControlledOperation` class is no longer imported top-level.
```

Comment:
Could this latter change break any plugins/developer code? I think it could be safe to keep the top-level import or have it go through a deprecation cycle as some developers might have used it.

Reply:
Searching all of the `PennyLaneAI` organization for `ControlledOperation` only returns things already modified by this PR.  Given the class has a big "Developer-only" qualification in the docstring, I'm not too worried about the change.  We should be able to change "developer-only" objects without deprecation cycles.